assets/js/redirect.js

```diff
@@ -96,5 +86,40 @@
 			}
 		}
 
+		/**
+		 * Validate URL for 'Redirect From' field.
+		 */
+		function validateUrl() {
+			currentRequest = $.ajax({
+				url: window.ajaxurl,
+				method : 'GET',
+				data : {
+					action: 'srm_validate_from_url',
+					from: fromRule.val(),
+					_wpnonce: $('#srm_redirect_nonce').val()
+				},
+				beforeSend : function() {
+					if ( currentRequest != null ) {
+						currentRequest.abort();
+					}
+				},
+				success: function( data ) {
+
+					// Remove loader.
+					if ( fromRule.hasClass( 'ui-autocomplete-loading' ) ) {
+						fromRule.removeClass( 'ui-autocomplete-loading' );
+					}
```

Comment:
```suggestion
					fromRule.removeClass( 'ui-autocomplete-loading' );
```

removeClass handles if the class doesn't exist so no need to check.

Reply:
Thank you for this. I've made this change now.